frontend/.eslintignore

```diff
@@ -7,3 +7,4 @@ public/lib
 Godeps
 @types
 dynamic-demo-plugin
+i18n-scripts
```

Comment:
We probably want to keep linting, just use a different eslintrc in the i18n-scripts folder with different rules.

Reply:
I moved in an eslintrc file; I'm still going through there and removing things that aren't necessary for this folder.